JDBC/DeleteBooks.java

```diff
@@ -0,0 +1,49 @@
+package com.metacube.jdbc.example;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+public class DeleteBooks {
+	public static void main(String[] args) throws IOException {
+		if(isBookDeleted())
+		System.out.println(DeleteBook.getNumberOfBooksDeleted()+" Book Deleted Successfully");
+		else
+			System.out.println("No book deleted");
+	}
+	/* execute query using prepared statement*/
+	/*method to get number of books deleted*/
+	private static boolean isBookDeleted() {
+		Connection con = null;
+		boolean isDeleted=false;
+		PreparedStatement ps = null;
+		ConnectionUtil conUtil = new ConnectionUtil();
```

Comment:
ACTION: What is the use of the "util" class u created if u have to do all these things in the functional class?